integration-tests/src/jvmTest/kotlin/ai/koog/integration/tests/MultipleLLMPromptExecutorIntegrationTest.kt

```diff
@@ -53,14 +60,19 @@ import kotlin.time.Duration.Companion.seconds
 class MultipleLLMPromptExecutorIntegrationTest {
 
     companion object {
-        private lateinit var testResourcesDir: File
+        private lateinit var testResourcesDir: Path
 
         @JvmStatic
         @BeforeAll
         fun setupTestResources() {
-            testResourcesDir = File("src/jvmTest/resources/media")
-            testResourcesDir.mkdirs()
-            assertTrue(testResourcesDir.exists(), "Test resources directory should exist")
+            testResourcesDir =
+                Paths.get(MultipleLLMPromptExecutorIntegrationTest::class.java.getResource("/media")!!.toURI())
+
+            if (!Files.exists(testResourcesDir)) {
+                Files.createDirectories(testResourcesDir)
+            }
+
+            assertTrue(Files.exists(testResourcesDir), "Test resources directory should exist")
```

Comment:
actually, you can use the same methods that you used with `File` -- e.g. `testResourcesDir.exists()` (but through another import)

and here this check is not needed, see the comment above